pkg/node/config_defaults.go

```diff
@@ -43,7 +43,7 @@ func NewDefaultComputeParam(storagePath string) ComputeConfigParams {
 
 var DefaultRequesterConfig = RequesterConfigParams{
 	JobDefaults: transformer.JobDefaults{
-		TotalTimeout: model.NoJobTimeout,
```

Comment:
maybe you should've just left it and migrated it to `models.NoJobTimeout` instead of doing this in many places. I am also thinking we should change this to `-1` to indicate no timeout instead of max value

Reply:
I think somewhere we have stated that `0` implies no timeout, wdyt of that pattern?

Reply:
as it stands, 0 means no timeout was defined and will be replaced by default value. I don't know how 0 timeout and 0 default timeout is handled in our system. Also 0 queue-timeout means no queueing enabled.

I'll be concerned about trying to change the meaning of 0 and -1 timeout as part of this change, but just having a `NoJobTimeout` const or var seemed a good approach to easily change things in the future

Reply:
Okay sounds good, will define a default constant and use everywhere.